packages/framework/src/schemas/steps/channels/in-app.schema.ts

```diff
@@ -18,6 +18,7 @@ const inAppOutputSchema = {
     avatar: { type: 'string', format: 'uri' },
     primaryAction: actionSchema,
     secondaryAction: actionSchema,
+    data: { type: 'object', additionalProperties: true },
```

Comment:
Looking also for a place where I can write the tests for this. I would appreciate it if someone could guide me.

Reply:
Schema changes in Framework are implicitly handled by tests covering the Framework's schema capability. Since we are just adding a new field, we don't need to add explicit tests for this change.